tests/migrations/helpers/test_update_legacy_filtered_positives_helper.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import numpy as np
+from migrations.helpers.update_legacy_filtered_positives_helper import (
+    v0_version_set,
+    legacy_mongo_samples,
+    get_cherrypicked_samples_by_date,
+    split_mongo_samples_by_version,
+)
+from crawler.constants import (
+    FIELD_PLATE_BARCODE,
+    FIELD_ROOT_SAMPLE_ID,
+    V0_V1_CUTOFF_TIMESTAMP,
+    V1_V2_CUTOFF_TIMESTAMP,
+)
+
+# ----- migration helper function tests -----
```

Comment:
Not urgent, but might be could to include some none happy path testing, e.g. when things throw, return empty arrays etc.